### DISCUSSION TITLE
I'm slow

### DISCUSSION THREAD

**Discussion by TooStupidToUseWindows** [dbeaver/dbeaver#22195] [Q&A]
I am just a simple user.   But I think I need to find some documentation to re-orient my thinking.  
I am trying to build a database of all my stock trades, (I have been coding for 50 years ).  I just need to parse CSV files and load them them into various tables.  I'm using Postgresql because I like the "type" and array support.  
So I have only one DB, only one server, but I getting a confused world of "connections", 'Schema", "Projects" , etc.  I'm not sure what happens, but I don't need all that stuff.. and it just gets in the way.  I realize that if someone is managing very large DB with multiple schemas and needs to test views etc with multiple connections, they need all those features.

I'll bring up a SQL editor.. have to some choices that don't make any sense to me.. .. but none that I know about .. just to connect to a server/db/schema.. but I now have about 20 different paths to get there . (I think.) 

Is there ANY documentation that makes it simple for a simple person to do a simple set of things like implement a simple DB?  
I truly appreciate the challenge DBeaver takes on to be all things for all DBs on all platforms, but I"m missing a very simple concept getting my scripts to just use one connection, one DB, one schema, etc.    It is never clear what is going to happen.

PS... Why am I allowed to close a SQL script tab without being prompted if I want to save.?  Or even it trying to save a copy before submitting it.. (like when the DB/connection crashes?

Sorry, I'm not bitching. I know it is me.

**Comment by LonwoLonwo** [comment]
Hello.

I can understand your feelings. Yes, DBeaver can be complicated.

Let's find some opportunities to make your work easier.

First, you can simplify your [navigator panel](https://github.com/dbeaver/dbeaver/wiki#database-navigator-panel).
Just change your navigator view to the [Simple](https://github.com/dbeaver/dbeaver/wiki/Simple-and-Advanced-View) or custom.

Here you can read more about data transfer:
https://github.com/dbeaver/dbeaver/wiki/Data-transfer
https://github.com/dbeaver/dbeaver/wiki/Data-migration


And about SQL Editor saving... Maybe these settings will be helpful:

![2024-01-04 15_41_16-Preferences](https://github.com/dbeaver/dbeaver/assets/45152336/b6b303bb-39a3-424f-8ec2-ac6a2717f6be)
